### PR TITLE
DOC: Expand README with history and acknowledgments

### PR DESCRIPTION
**Pull request by dzenanz**

### CODE REVIEW

**Comment by dzenanz** [comment]
Longer term TODO: update the code to mirror the one [in Slicer](https://github.com/Slicer/Slicer/blob/master/Modules/Loadable/Segmentations/Logic/vtkImageGrowCutSegment.cxx).

@lassoan @allywarner @ljzhu and @harveerar please review.

File: README.rst [hunk 1]
```diff
@@ -15,6 +15,41 @@ ITKGrowCut
...
+
+
+Acknowledgements
+----------------
+
+This software was developed in part by the Center for Integrative Biomedical Computing (CIBC) and the
+`Scientific Computing and Imaging (SCI) Institute <https://www.sci.utah.edu/cibc>`_ and
+`Kitware <https://www.kitware.com>`_.
+
+Support came from the National Institute of General Medical Sciences of the National Institutes of Health
```
Comment (jcfr) [inline README.rst]:
```suggestion
Support came from the National Institute of General Medical Sciences (NIGMS) of the National Institutes of Health (NIH)
```

File: README.rst [hunk 2]
```diff
@@ -15,6 +15,41 @@ ITKGrowCut
...
+<https://github.com/Slicer/Slicer/blob/master/Modules/Loadable/Segmentations/Logic/vtkImageGrowCutSegment.cxx>`_.
+
+
+In this remote module we are providing and ITK-based implementation of the efficient algorithm by Zhu et al.
+
+
+Acknowledgements
+----------------
+
+This software was developed in part by the Center for Integrative Biomedical Computing (CIBC) and the
```
Comment (jcfr) [inline README.rst]:
```suggestion
This software was developed in part by the Center for Integrative Biomedical Computing (CIBC), the
```

File: README.rst [hunk 3]
```diff
@@ -15,6 +15,41 @@ ITKGrowCut
...
+ | International Conference on Medical Image Computing and Computer Assisted Intervention (MICCAI),
+ | Interactive Medical Image Computing Workshop, 2014
+
+Zhu et al. also provided an open source implementation as a Slicer plugin, `based on VTK
+<https://github.com/ljzhu/FastGrowCut>`_.
+Since then, their implementation was integrated into Slicer, `refactored and improved
+<https://github.com/Slicer/Slicer/blob/master/Modules/Loadable/Segmentations/Logic/vtkImageGrowCutSegment.cxx>`_.
+
+
+In this remote module we are providing and ITK-based implementation of the efficient algorithm by Zhu et al.
```
Comment (jcfr) [inline README.rst]:
```suggestion
In this remote module we are providing an ITK-based implementation of the efficient algorithm by Zhu et al.
```

**Comment by jcfr** [comment]
> Longer term TODO: update the code to mirror the one in Slicer.

What do you think of creating an issue to document this ?

**Comment by dzenanz** [comment]
Issue #3 created to track the todo.

**Comment by lassoan** [comment]
This looks good to me.

Integrating the improvements that we did in the Slicer implementation (#3) is very important, as they provide significant performance and robustness improvements and add essential feature such as masking. If these improvements are integrated then Slicer can switch to use this implementation.

**Comment by dzenanz** [comment]
> ... then Slicer can switch to use this implementation

That is good to hear! With GrowCut as an ITK module, more people can benefit from it. The effort was spearheaded by @allywarner for [Seg3D](https://github.com/SCIInstitute/Seg3D/pull/394) project.

**Review by thewtex — APPROVED** [review]
:+1:

**Pull request merged**